Initial Commit v0.1.0

dmaloneycalu opened the pull request:
Copied and tidied up code from existing project. Created a roadmap for where it should go.

After this is successfully merged/tagged, we can release it through Bower (allowing us to reference it as just a version number and not its github address). Any and all thoughts appreciated.

@StoicJester @jrit @Tathanen
@scottferg

scottferg commented:
Needs an open source license and the proper heading on source files. MIT or BSD will suffice.

Tathanen commented on svg-radial.js:
```diff
@@ -0,0 +1,51 @@
+                                '<div class="piece"></div>' +
+                            '</div>' +
+                            '<div class="transclude" ng-transclude></div>' +
+                        '</div',
```
Missing >?

Think about using a template file instead, too.

Tathanen commented on less/svg-radial.less:
```diff
@@ -0,0 +1,87 @@
+    -o-transition:      -o-transform @time ease-out;
+    -ms-transition:     -ms-transform @time ease-out;
+    transition:         transform @time ease-out;
+}
```
Wow that was a lot of variables.

jrit commented on svg-radial.js:
```diff
@@ -0,0 +1,51 @@
+"use strict";
```
maybe move this out of global scope for concat friendliness

Tathanen replied:
Seconded.

Tathanen commented on svg-radial.js:
```diff
@@ -0,0 +1,51 @@
+                    setRotation( ".piece:first-child", angle1 );
+                    setRotation( ".piece:last-child", angle2 );
+
+                    element.find( ".radial" ).toggleClass( "gtFifty", ( percent > 50 ) );
```
I believe finding with selectors only works if you include full jQuery.  So if you wanna do that, make jQuery a dependency in bower.json.

StoicJester replied:
yup, limited to tag name finding

dmaloneycalu replied:
Good catch, will include that in the bower file. Adding it to the roadmap to remove that dependency by v1.

jrit commented on less/svg-radial.less:
```diff
@@ -0,0 +1,87 @@
+        @radial-progress-border-size -
+        @radial-progress-border-size -
+        @radial-progress-transclude-buffer -
+        @radial-progress-transclude-buffer;
```
some mixed indents

StoicJester replied:
Those lines are indented because they are actually being subtracted from the line above

jrit replied:
so they are, we usually have the operators at the start of the lines and was "operator blind"

Tathanen commented on svg-radial.js:
```diff
@@ -0,0 +1,51 @@
+    .directive( "radialProgress", function ()
+    {
+        return {
+            restrict: "E",
```
Any reason in particular that you want it restricted?

dmaloneycalu replied:
There's no particular reason for this yet. It was just how it was included in our project at the time. All of this is mostly copy/paste.

Tathanen commented on less/svg-radial.less:
```diff
@@ -0,0 +1,87 @@
+        border-radius: 50%;
+        overflow: hidden;
+
+
```
Superfluous second blank line.

dmaloneycalu commented:
@Tathanen @jrit 

- Added jQuery as dependency
- Closed the div tag
- Move the `use strict` statement

@scottferg 

- Added `license.md` and a license block to the two source files

scottferg commented:
All caps, `LICENSE.md` then LGTM

scottferg commented on license.md:
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Vokal
```
Vokal LLC

dmaloneycalu commented:
@jrit @Tathanen @StoicJester Any last notes?

@scottferg completed the LLC changes

dmaloneycalu commented:
Bump :+1: :-1:

Tathanen commented:
LGTM
